lib/core/Screens/chatScreen.dart

```diff
@@ -1,22 +1,28 @@
+import 'package:chatapp/UIwidgets/massageWidget.dart';
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:flutter/painting.dart';
 import 'package:flutter/rendering.dart';
 import 'package:flutter/widgets.dart';
 import 'package:firebase_auth/firebase_auth.dart';
 import 'package:cloud_firestore/cloud_firestore.dart';
+final _firestore =FirebaseFirestore.instance;
+late User signInUser;  //to save the user mail that will send
+
 class ChatScreen extends StatefulWidget {
+
   static const String ScreenRoute ="ChatScreen" ;
   const ChatScreen({super.key});
 
+
   @override
   State<ChatScreen> createState() => _ChatScreenState();
 }
 
 class _ChatScreenState extends State<ChatScreen> {
-  final _firestore =FirebaseFirestore.instance;
+ final massageTextcontrolar=TextEditingController();
   final _auth=FirebaseAuth.instance;
-  late User signInUser;  //to save the user mail that will send
```

Comment:
i had comment this to de remember that the request didn't allowed when variable is late in the class this will be delcer